docs/notebooks/bsync_examples/Small-Office-GHG.ipynb

```diff
@@ -0,0 +1,1435 @@
+    "utilities += ng_ut"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d149e7ad",
+   "metadata": {},
+   "source": [
+    "#### ResourceUses and TimeSeries Data\n",
+    "Now that we have a current building measured scenario, we want to declare energy and monthly billing data. Per Std 211 6.1.2.1, a minimum of 12 months (preferably up to 3 years) of energy use data is required. The mechanical system of the small office prototype is a heatpump air handler with natural gas backup. We know that Missoula is cold, so likely it will use natural gas backup at some point during its operation. We run an example simulation to get estimates for this, which come out as follows:\n",
+    "\n",
+    "| Resource Type | Jan | Feb | Mar | Apr | May | Jun | Jul | Aug | Sep | Oct | Nov | Dec |\n",
+    "| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |\n",
+    "| Electricity (kWh) | 6792.89 | 5841.75 | 6025.19 | 4985.3 | 5184.04 | 5358.55 | 5755.67 | 5981.78 | 5401.94 | 5225.84 | 5672.15 | 6291.63 |\n",
+    "| Natural Gas (MMBtu) | 5.7 | 4.01 | 0.58 | 0.4 | 0.02 | 0 | 0 | 0 | 0 | 0.01 | 0.36 | 6.08 | 17.16 |\n",
+    "\n",
+    "In BuildingSync land, we need to declare an resource use for each resource type. Standard, allowable enumerations exist for this already. We do this as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "309a0516",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "all_ru = bsync.ResourceUses()\n",
+    "\n",
+    "# create a resource use for electricity, units of kWh, all end uses\n",
+    "# we also add info about the 'peak' units\n",
+    "# we also connect it up to a utility\n",
+    "elec_ru = bsync.ResourceUse(\n",
+    "    bsync.EnergyResource('Electricity'),\n",
+    "    bsync.ResourceUseNotes(\"This is required for L1 to document irregularities in monthly energy patterns (Std 211 6.1.2.1.j). No irregularities found.\"),\n",
+    "    bsync.ResourceUnits('kWh'),\n",
+    "    bsync.PeakResourceUnits('kW'),\n",
+    "    bsync.EndUse('All end uses'),\n",
+    "    bsync.UtilityIDs(\n",
+    "        bsync.UtilityID(IDref=elec_ut['ID'])\n",
+    "    )\n",
+    ")\n",
+    "\n",
+    "# given the above, we add the annual totals\n",
+    "elec_ru += bsync.AnnualFuelUseNativeUnits(68516.73)\n",
+    "elec_ru += bsync.AnnualFuelUseConsistentUnits(234.) #convert to MMBTU\n",
+    "elec_ru += bsync.AnnualPeakNativeUnits(21.12) # kW as specified above\n",
+    "elec_ru += bsync.AnnualPeakConsistentUnits(21.12) # already in kW, same as above\n",
+    "elec_ru += bsync.AnnualFuelCost(5304.) # $28 monthly + $0.0725/kwh per https://missoulaelectric.com/member-care/billing-payment/rates/\n",
+    "\n",
+    "# create a resource use for electricity, units of kWh, all end uses\n",
+    "# we also add info about the 'peak' units\n",
+    "# we also connect it up to a utility\n",
+    "ghg_linked_ids = bsync.EmissionsLinkedTimeSeriesIDs()\n",
```

Comment:
@JieXiong9119  should this section (lines 708-711) be included similar to how it's done for lines 747-748?